### PR TITLE
chore: set up SUPPORT and ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by tsvetomir**
Attempting to shed some light on the role of the different support channels.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+-->
+
+## What is the motivation / use case for changing the behavior?
+<!-- Describe the motivation or the concrete use case. -->
+
+## Environment
+<!-- Check whether this is still an issue in the most recent package versions -->
+
+Package versions:
+<!-- Paste the @progress dependencies from your package.json file -->
```
Comment (rkonstantinov) [inline .github/ISSUE_TEMPLATE.md]: Maybe we should ask for `@angular` version too?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  2. Open it in Plunker by using the **OPEN AS PLUNKER** button.
+  3. Modify the plunk, so that the issue is reproducible in it. Try to include the minimum possible amount of code.
+  4. Save the plunk by using the blue **Save** button at the top.
+  5. Copy the link from the address bar of the browser.
+-->
+
+## What is the motivation / use case for changing the behavior?
+<!-- Describe the motivation or the concrete use case. -->
+
+## Environment
```
Comment (rusev) [inline .github/ISSUE_TEMPLATE.md]: We should have `typescript` and `angular` sections as well.

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+## How do I&hellip;?
```
Comment (rusev) [inline SUPPORT.md]: `I&hellip;?` :)

**Comment by tsvetomir** [comment]
@danielkaradachki AFAIK there's only a link to the SUPPORT file in the yellow banner. We can shorten it if it is indeed included on the page.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (tsvetomir) [inline .github/ISSUE_TEMPLATE.md]: Done

File: SUPPORT.md (hunk 3 above)
Comment (tsvetomir) [inline SUPPORT.md]: HTML-fu for ...

image

**Comment by rusev** [comment]
@tsvetomir @danielkaradachki Indeed, the banner will contain a link to the support file:
>... Then, whenever someone creates an issue in your repository, they will see a link to your project's SUPPORT file.

https://help.github.com/articles/adding-support-resources-to-your-project/

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (ggkrustev) [inline .github/ISSUE_TEMPLATE.md]: Would the `angular-cli` version be helpful for debugging? Just wondering :)

File: SUPPORT.md (hunk 3 above)
Comment (gyoshev) [inline SUPPORT.md]: Stands for **h**orizontal **ellip**sis&hellip; brother of `&vellip;` the vertical ellipsis

&vellip;

In case you were wondering :)

**Pull request merged**